phpunit-dama-doctrine.xml.dist

```diff
@@ -11,7 +11,11 @@
         <ini name="error_reporting" value="-1"/>
         <env name="KERNEL_CLASS" value="Zenstruck\Foundry\Tests\Fixtures\Kernel"/>
         <env name="USE_DAMA_DOCTRINE_TEST_BUNDLE" value="1"/>
-        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0&amp;max[direct]=0&amp;quiet[]=indirect&amp;quiet[]=other"/>
+        <!--
+            max[self]=1 represents "The Zenstruck\Foundry\BaseFactory::new() method is considered final" deprecation.
+            It could be removed in 2.0
+        -->
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=1&amp;max[direct]=0&amp;quiet[]=indirect&amp;quiet[]=other"/>
```

Comment:
I tried to use the baseline or ignore files features of the bridge, but it still fails with sf 5.4